pkg/cluster/check/default.go

```diff
@@ -43,6 +43,12 @@ func DefaultClusterChecks() []ClusterCheck {
 				return ApidReadyAssertion(ctx, cluster)
 			}, 2*time.Minute, 5*time.Second)
 		},
+		// wait for kubelet to be healthy on all
+		func(cluster ClusterInfo) conditions.Condition {
```

Comment:
additional check for kubelet, should make sense before we check node status (so that we don't see stale nodes)